tests/api.rs

```diff
@@ -119,3 +119,42 @@ fn get_identifier() {
     let id = fw.get_identifier().unwrap();
     assert_ne!(Vec::from(id), vec![0u8; 64]);
 }
+
+#[cfg_attr(not(has_sev), ignore)]
+#[test]
+fn snp_platform_status() {
+    let mut fw = Firmware::open().unwrap();
+    let status = fw.snp_platform_status().unwrap();
+
+    println!(
+        "Platform status ioctl results:\n
+              version (major, minor): {}.{}\n
+              build id: {}\n
+              guests: {}\n
+              tcb version: {}\n
+              state: {}\n",
+        status.build.version.major,
+        status.build.version.minor,
+        status.build.build,
+        status.guests,
+        status.tcb_version,
+        status.state
+    );
+}
+
+#[cfg_attr(not(has_sev), ignore)]
+#[test]
+fn snp_ext_config() {
+    let mut fw = Firmware::open().unwrap();
+
+    fw.snp_set_ext_config(0, 0, 0).unwrap();
+
+    let config = fw.snp_get_ext_config().unwrap();
+    println!(
+        "Get platform config results:\n
+        config_address: {}\n
+        certs_address: {}\n
+        certs_len: {}",
+        config.config_address, config.certs_address, config.certs_len
+    );
```

Comment:
Can you further explain?

Reply:
For example, the reset command is gated behind a hardware feature called `dangerous_hw_tests` which must be manually enabled because it clears the persistent state of the platform. If this happens while CI is running, some of the tests will spuriously fail.

Do you anticipate running this command to have similar side effects against other VMs?

Reply:
It seems that randomly setting new configuration values via `snp_set_exit_config` could affect other VMs (at the very least, ones that are running SNP) on a system. I'll add a dangerous_hw_tests feature.